src/main/resources/db/migration/V2023.09.06.0001__add_prison_id_fields_to_goal.sql

```diff
@@ -0,0 +1,8 @@
+ALTER TABLE goal ADD COLUMN created_at_prison VARCHAR(20);
+ALTER TABLE goal ADD COLUMN updated_at_prison VARCHAR(20);
+
+UPDATE goal SET created_at_prison = 'MDI';
+UPDATE goal SET updated_at_prison = 'MDI';
```

Comment:
This is the prisonId of the prisoners in the list we provided here: https://github.com/ministryofjustice/hmpps-education-and-work-plan-ui/blob/main/server/views/pages/index.njk